components/home/Testimonials.vue

```diff
@@ -1,121 +1,70 @@
 <template>
-  <div class="bg-magnolia">
-    <div class="container">
-      <!-- Desktop View -->
-      <div class="d-none d-md-grid testimonials-section">
-        <div>
-          <h2 class="testimonials-heading">Testimonials</h2>
-          <div class="users">
-            <div
-              class="d-flex flex-column users-wrapper"
-              :style="usersWrapperStyling"
-            >
-              <div
-                v-for="(testimonial, idx) in testimonials"
-                :key="testimonial.id"
-                class="user d-flex align-items-center"
-                :class="{ active: active === testimonial.id }"
-                @click="updateActive(testimonial.id)"
-              >
-                <nuxt-img
-                  :src="`/testimonials/${testimonial.picture}`"
-                  :alt="testimonial.picture"
-                  loading="lazy"
-                />
-                <div>
-                  <h6 class="name">{{ testimonial.user }}</h6>
-                  <span class="designation">{{ testimonial.designation }}</span>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div class="testimonial-content-wrapper">
-          <p class="mb-3">{{ activeTestimonial.content }}</p>
-          <div class="d-flex justify-content-end">
-            <img
-              :src="require(`@/assets/images/testimonials/${activeTestimonial.logo}`)"
-              :alt="activeTestimonial.logoAlt"
-            />
-          </div>
-          <nuxt-img
-            src="/testimonials/quotation.svg"
-            class="quotation-marks"
-            alt="Quotation marks"
-            loading="lazy"
-          />
-        </div>
+  <div class="container my-5 py-5">
+    <div class="d-flex justify-content-between align-items-center my-5">
+      <h2 class="section__heading position-relative">
+        What our Customers say
         <nuxt-img
-          src="/testimonials/quote.svg"
-          class="quote-illustraion"
-          alt="Quote illustration"
+          src="UI Block/Vector 5.svg"
```

Comment:
Same in all src attribute, try to use path with @/ .

Reply:
@hanzalakhan809 we are using the nuxt-img where the base path has already been assigned.